### ISSUE TITLE
 ERROR: 42601: LIMIT #,# syntax is not supported at character 1080

### ISSUE THREAD

**Issue by pyhalov** [examus/moodle-availability_examus#14]
When trying to use plugin with Moodle on PostgreSQL, we see the following issue:
```
ERROR: 42601: LIMIT #,# syntax is not supported at character 1080
HINT: Use separate LIMIT and OFFSET clauses.
LOCATION: base_yyparse, gram.y:11185

STATEMENT: SELECT e.id id, e.timemodified timemodified, timescheduled, u.firstname u_firstname, u.lastname u_lastname, u.email u_email, u.id userid, e.status status, review_link, cmid, courseid FROM mdl_availability_examus e LEFT JOIN mdl_user u ON u.id=e.userid WHERE 1 AND e.timemodified <= $1 AND courseid
Jul 6 20:38:32 pgsql postgres[12254]: [8-5] IN(150,22,17,112,183,184,33,49,179,225,8,121,122,207,77,82,88,72,124,89,229,193,29,222,80,180,143,96,105,6,36,117,73,30,227,115,79,101,171,192,199,120,62,154,161,67,141,152,135,223,166,103,212,170,113,91,137,60,108,164,86,156,157,38,20,2,10,32,43,37,148,51,46,31,204,44,228,47,7,202,18,109,118,92,130,81,195,13,98,71,70,127,54,3,172,74,185,173,64,196,14,155,123,106,144,52,53,181,178,75,163,61,139,25,203,151,198,194,136,119,174,26,219,168,126,201,69,5,99,149,129,147,39,84,197,15,66,65,19,35,21,217,131,107,97,182,146,142,162,145,216,215,63,90,95,167,55,9,200,205,24,230,220,132,210,134,111,1,57,93,56,102,191,94,100,169,110,221,177,206,140,87,85,116,114,4,176,59,27,158,231,226,125,211,41,133,68,48,189,28,224,11,50,190,159,153,78,209,160,58,104,83,188,165,186,187,208,175,23) LIMIT 0,30
```

**Comment by pyhalov** [comment]
Likkely, https://github.com/examus/moodle-availability_examus/blob/c38fe6c3c9d399699473f1c0db6365edc418fe29/classes/log.php#L101 should be fixed.

**Comment by MaksimBurnin** [comment]
Fixed in 2b7ae1395a5a5044de579778c3eca346908ca52a
